latch/ldata/_transfer/node.py

```diff
@@ -2,18 +2,26 @@
 from typing import Dict, List, TypedDict
 
 import graphql.language as l
-from latch_sdk_gql.execute import execute
 from latch_sdk_gql.utils import _name_node, _parse_selection
 from typing_extensions import TypeAlias
 
 from latch.ldata.type import LDataNodeType
-from latch_cli.utils.path import get_path_error, normalize_path
+from latch_cli.utils.path import normalize_path
+
+from .utils import query_with_retry
 
 AccId: TypeAlias = int
 
 
-class LatchPathNotFound(RuntimeError):
-    pass
+class LatchPathError(RuntimeError):
+    def __init__(self, message: str, remote_path: str = None, acc_id: str = None):
+        super().__init__(message)
+        self.message = message
+        self.remote_path = remote_path
+        self.acc_id = acc_id
```

Comment:
unused?

Reply:
its being used when `get_path_error` gets called in `latch cp`